meinberlin/assets/scss/components/_filter_bar.scss

```diff
@@ -235,11 +235,13 @@ $divider-margin: 1.25rem;
 
 .filter-bar__search {
     display: inline;
-    width: calc(100% - 63px);
+    width: calc(100% - 37.1875px);
     margin-bottom: 2 * $spacer;
+    border-top-left-radius: 0.3em !important;
```

Comment:
will look into this one again, to avoid `!important`. This gets overwritten by another class, will check if can be removed there.

Reply:
okay, doublechecked it and saw that another class is overwriting `border-radius: 0`. This class is widely used so leaving the `!important` for now, if that is fine @phillimorland ?

Reply:
I'm thinking maybe we should re-write our search inputs to use bootstrap mark up 
```
<div class="input-group mb-3">
  <input type="text" class="form-control" placeholder="search" aria-label="search" aria-describedby="id-search">
  <div class="input-group-append">
    <button class="btn btn-outline-secondary" type="button" id="id-search">Button</button>
  </div>
</div>
```
what do you think?

Reply:
yes that makes sense, i will have a look and give it a try @phillimorland